ddmrp/models/stock_buffer_profile.py

```diff
@@ -62,3 +64,11 @@ def _compute_name(self):
         help="When activated, the recommended quantity will be maxed at "
         "the quantity available in the replenishment source location.",
     )
+    distributed_reschedule_max_proc_time = fields.Float(
```

Comment:
Should not we put an `onchange` depending on `item_type` to reset `distributed_reschedule_max_proc_time` to 0 as soon as `item_type != 'distributed'`? Or add a check on this where this value is used? If we switch to the type `manufactured` we are still using the existing value, don't know if it's what we want here.

Reply:
Added a check where it's used